src/factory_sim/config/config.yaml

```diff
@@ -145,20 +141,11 @@ objectives:
   # Specify source folder for objectives
   # [Required]
   objective_library_paths:
-    core_objectives:
-      package_name: "moveit_pro_objectives"
-      relative_path: "objectives/core"
-    motion_objectives:
-      package_name: "moveit_pro_objectives"
-      relative_path: "objectives/motion"
-    perception_objectives:
-      package_name: "moveit_pro_objectives"
-      relative_path: "objectives/perception"
-    sim_objectives:
-      package_name: "fanuc_sim"
+    core:
+      package_name: "factory_sim"
```

Comment:
I talked with Ezra and we are going to write the migration guide pointing people to the new "core objectives" as part of 8.0 migration to VFC based servoing.